### DISCUSSION TITLE
Accesibilty - contrast on buttons

### DISCUSSION THREAD

**Discussion by ericvd-ucb** [kevinlin1/just-the-class#32] [General]
I just ran my site through an accessibility checker
https://www.econ148.org/sp23/
The only problem it came up with was contrast on the buttons

Is there a way to configure to an accessible set of buttons?

**Comment by kevinlin1** [comment]
Hey Eric: Yes, you can customize the colors of your buttons beyond the options provided by Just the Class! Here's an example that you can add to your `_sass/custom/custom.scss` file to define a button class with white text and a [dark purple](https://just-the-docs.com/docs/utilities/color/) background.

```scss
.btn-dark-purple {
  @include btn-color($white, $purple-300);
}
```

You can view all the default button color definitions in the [upstream project's `_sass/button.scss` file](https://github.com/just-the-docs/just-the-docs/blob/main/_sass/buttons.scss#L99-L113).

@mattxwang has the Just the Docs (upstream) project considered this change?

**Reply by mattxwang** [reply]
Thanks for tagging me in!

Improving the theme defaults wrt accessibility has been in the backlog in the while, though I haven't had a chance to get around to it yet 😓 . This is a great reminder though, I'll likely submit a PR over the weekend (I have a few other related changes I'd like to make as well wrt contrast).

**Comment by ericvd-ucb** [comment]
wow Kevin you rock - thanks for the quick reply - 

At some point it would be cool to make a set of accessibility contrasting buttons that others could use - this should make that easy
